app/lib/proof_provider/keybase/config_serializer.rb

```diff
@@ -34,7 +34,7 @@ def description
   end
 
   def username
-    { min: 1, max: 30, re: '[a-z0-9_]+([a-z0-9_\.-]+[a-z0-9_]+)?' }
+    { min: 1, max: 30, re: '[a-z0-9_]+([a-z0-9_.-]+[a-z0-9_]+)?' }
```

Comment:
I feel like if this is causing issues in their JSON parsing or in our JSON serialization, that's a bigger issue and we shouldn't paper over it by removing this slash.